lunavl/sdk/luna_faces.py

```diff
@@ -188,6 +188,17 @@ def mask(self) -> Mask:
             self._mask = self.estimatorCollection.maskEstimator.estimate(self.warp)
         return self._mask
 
+    @property
+    def maskV2(self) -> Mask:
```

Comment:
""There should be one-- and preferably only one --obvious way to do it.""
change property mask.

Reply:
I removed this and added it in mask property